exp/api/v1alpha3/awsmachinepool_types.go

```diff
@@ -67,6 +67,11 @@ type AWSMachinePoolSpec struct {
 	// This field must match the provider IDs as seen on the node objects corresponding to a machine pool's machine instances.
 	// +optional
 	ProviderIDList []string `json:"providerIDList,omitempty"`
+
+	// The amount of time, in seconds, after a scaling activity completes before another scaling activity can start.
+	// +kubebuilder:default=300
+	// +optional
```

Comment:
Not sure having it optional but defaulting to 300 results in anything useful. Either remove the optional flag, or move defaulting logic to the controller.

Reply:
Maybe also change the type to the more Kubernetes style of using time.ParseDuration, so users can specify "5m" etc... That would hint towards removing the optional flag, and setting the default.

Reply:
Thanks for the suggestion...you mean change the type to string and then in the controller somewhere use time.ParseDuration and use Seconds() to get the value in float64. Now I am having a bit of trouble converting that to *int64 because that's what the aws lib expects

Reply:
converting to int64 is fine. You'll need to use metav1.Duration